bin/endorse-bayes

```diff
@@ -11,23 +11,27 @@ ENDORSE_VENV_BAYES="${ENDORSE_SRC_ROOT}/venv_bayes"
 run=false
```

Comment:
This seems like a stub, having main script in endorse-bukov or endorse-bayes-sensitivity. Describe purpose.